### PR TITLE
[MI-3573] Add issue template

### PR DESCRIPTION
**Pull request by ayusht2810**
#### Summary
Add issue template

#### Ticket Link
#211

### CODE REVIEW

**Review by raghavaggarwal2308 — APPROVED** [review]

File: docs/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+name: Bug report
+about: Create an to report bug
```
Comment (Kshitij-Katiyar) [inline docs/ISSUE_TEMPLATE/bug_report.md:3]: Is this correct ?

File: docs/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+title: 'Bug: '
+labels: Bug, Triage
+assignees: ''
```
Comment (Kshitij-Katiyar) [inline docs/ISSUE_TEMPLATE/bug_report.md:6]: Please specify here how we need to add assignees. Like @username or just username etc

File: docs/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+## Actual behaviour
+
+A clear and concise description of what actually happened.
+
+## Screenshots (optional)
+
+If applicable, add screenshots or a screen recording to elaborate on the problem.
+
+## Mattermost Version and Platform
```
Comment (Kshitij-Katiyar) [inline docs/ISSUE_TEMPLATE/bug_report.md:29]: Maybe ?
```suggestion
## Mattermost Version and other Platforms
```

File: docs/ISSUE_TEMPLATE/enhancement.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+about: Suggest a new feature
+title: 'Feature Idea: '
+labels: Enhancement, Triage
+assignees: ''
+
+---
+
+## Summary
+
+What the new feature is.
```
Comment (Kshitij-Katiyar) [inline docs/ISSUE_TEMPLATE/enhancement.md:12]: Maybe
```suggestion
Describe the new feature.
```

File: docs/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+ - Mattermost Version: [e.g. v7.8.10]
+ - Browser and OS: [e.g. Chrome on Mac, Edge on Windows]
+
+## Additional context (optional)
+
+Add any other context about the problem here, and any notes about the severity:
+- Timestamp during which the issue occurred.
+- Mattermost username or user ID
+- MS Teams username or user ID
+- Platform from where the message originated
```
Comment (manojmalik20) [inline docs/ISSUE_TEMPLATE/bug_report.md:40]: I think this should not be optional.

**Review by manojmalik20 — APPROVED** [review]

**Review by manojmalik20 — APPROVED** [review]

**Pull request merged**